lib/ddtrace/contrib/qless/tracer_cleaner.rb

```diff
@@ -0,0 +1,31 @@
+module Datadog
+  module Contrib
+    module Qless
+      # Shutdown Tracer in forks for performance reasons
+      module TracerCleaner
+        def around_perform(job)
+          return super unless datadog_configuration && tracer
+
+          tracer.shutdown! if forked?
```

Comment:
Do you think we'd be able to exercise this fork clean up in a test case?

Reply:
I tried adding tests for the forked path but I couldn't get the forked process working correctly in tests (we had a conversation about testing forked code back in https://github.com/DataDog/dd-trace-rb/issues/1053).  I've manually tested it and the tracer is shutdown.

Reply:
I'll set up some time so we help you with this. We'll try to write a forking test and see what issues we encounter.